kits/melon/package.json

```diff
@@ -31,7 +33,9 @@
     "eth-ens-namehash": "^2.0.8"
```

Comment:
We should add `@aragon/apps-agent@1.0.1`, and pin the other contract dependencies here.